spec/factories/case_transitions.rb

```diff
@@ -327,6 +327,14 @@
 
   # Offender SAR specific transitions below - note use of responder
```

Comment:
Does this comment need updating now, as it only refers to Offender SARs and not Offender SAR Complaints?